docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<p>One other comment is that we support <code>rzz(theta)</code> for <code>theta</code> in the range of [0, pi].   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Got it, is that also true for the <code>rx(theta)</code> gate? </p>

Reply:
No Rx supports full range.

Reply:
The range of rzz is `[0, pi/2]`.